certbot/certbot/display/util.py

```diff
@@ -46,26 +47,187 @@
 """Display boundary (alternates spaces, so when copy-pasted, markdown doesn't interpret
 it as a heading)"""
 
-def _wrap_lines(msg):
-    """Format lines nicely to 80 chars.
 
-    :param str msg: Original message
+# This class holds the global state of the display service to store, in a more
+# consistent way than the "global" keyword.
```

Comment:
```suggestion
# This class holds the global state of the display service to avoid the use of
# the "global" keyword when trying to modify it which can be error prone. See
# https://github.com/certbot/certbot/pull/8835/files#r644313793 for more info.
```